src/components/user-block/react/UserBlock.tsx

```diff
@@ -38,7 +38,7 @@ interface IUserBlockProps extends IGenericProps {
     /* User name. */
     name?: string;
     /* Orientation. */
-    orientation?: ORIENTATIONS;
+    orientation?: Orientations;
```

Comment:
`Orientation` type instead of `Orientations` enum ?

Reply:
done